tests/e2e/default/db/initial-replication.wdio-spec.js

```diff
@@ -105,13 +105,12 @@ const refreshAndWaitForAngular = async () => {
 
 describe('initial-replication', () => {
   before(async () => {
+    await utils.stopSentinel();
```

Comment:
We start sentinel again in the `after` but we don't wait for it to process the 2k docs. Do you think this will impact other tests or can we just safely leave it running in the background?

Reply:
The code we run in the after is:
```
 await sentinelUtils.skipToSeq();
 await utils.startSentinel();
```

`skipToSeq` will change the transitions-seq value to the current update_seq of the database. so we're not waiting for sentinel to process docs here because the queue should be empty. 
Are you worried about processing deletes?

Reply:
Hmm no in retrospect I think it's fine. Thanks for explaining!